storage/kvstore.go

```diff
@@ -489,3 +489,29 @@ func (s *store) delete(key []byte) {
 	}
 	s.currentRev.sub += 1
 }
+
+// appendMarkTombstone appends tombstone mark to the backend key.
```

Comment:
I feel calling Tombstone a type is better than a mark. 

Some context: a revision might be a normal kv or a tombstone of a kv.

/cc @jonboulle